src/app/Components/ComponentsSamples/ContextMenus/ContextMenuSamples.xaml

```diff
@@ -0,0 +1,163 @@
+﻿<dui:ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                 xmlns:dui="http://dips.com/mobile.ui"
+                 xmlns:contextMenu="clr-namespace:Components.ComponentsSamples.ContextMenu"
+                 x:Class="Components.ComponentsSamples.ContextMenus.ContextMenuSamples"
+                 Padding="15">
+    <ContentPage.BindingContext>
+        <contextMenu:ContextMenuSamplesViewModel />
+    </ContentPage.BindingContext>
+    <StackLayout>
+        <!-- Single item -->
+        <dui:Button Text="Single item"
+                    dui:ContextMenuEffect.HasContextMenu="True">
```

Comment:
Vi kunne kanskje ha droppet HasContextMenu property? Det holder muligens å sette ItemsSource for å forstå at man skal ha en context menu?